### PR TITLE
Fix eurostat edu enrollment StatVar MCF typo by removing the file

### PR DESCRIPTION
**Pull request by tjann**
There was a typo in the StatVar MCF in GitHub but not in g3. I removed the StatVar MCF here in favor of the ones in g3. Here are some screenshots for validation (I resolved without extra StatVar, just relying on g3):

image

image

### CODE REVIEW

File: scripts/world_bank/output/WorldBank_StatisticalVariables.mcf [hunk 1]
```diff
@@ -0,0 +1,557 @@
+
+Node: dcid:WorldBank/SE_TER_CUAT_BA_FE_ZS
```
Comment (pradh) [inline scripts/world_bank/output/WorldBank_StatisticalVariables.mcf:2]: Is this change intentional?

Comment (tjann) [reply scripts/world_bank/output/WorldBank_StatisticalVariables.mcf]: No, reverted.

**Review by pradh — APPROVED** [review]
